kernel/selinux/Makefile

```diff
@@ -2,6 +2,13 @@ obj-y += selinux.o
 obj-y += sepolicy.o
 obj-y += rules.o
 
+ifeq ($(shell grep -q " current_sid(void)" $(srctree)/security/selinux/include/objsec.h; echo $$?),0)
+ccflags-y += -DHAS_CURRENT_SID
```

Comment:
please prefix this MACRI with `KSU_` to avoid potential conflicts

Reply:
https://github.com/tiann/KernelSU/pull/402/commits/47d21ee22a9f6b707246f2a06ec69814941d837f prefixed with `KSU_COMPAT_`